misc/org.projecthamster.hamster-gtk.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=hamster-gtk
```

Comment:
I prefer non-generic names for software but if an organization creates an app with multiple different frontends, it makes sense for all of them to share the name. If the QT frontend is developed by a third party not connected to projecthamster.org then using a different name might be okay.